lpd/tests/test_qualitative_data_analysis.py

```diff
@@ -51,3 +54,49 @@ def test_make_document(self, answers, expected_document):
         and performs stemming for inflected forms of 'teach' root.
         """
         self.assertEqual(qda.make_document(answers), expected_document)
+
+    @data(
+        (
+            [
+                'Teaching is awesome, I love it.',
+                'I have many teachers in my family, I thought of becoming one.',
+                'I would love to remove gaps among marginalized social groups.',
+                (
+                    'I need to improve my knowledge and skill, having worked in different places in the world,'
+                    'I\'ve gathered better understanding of different cultures.'
+                )
+            ],
+            {
+                'kc_id_1_set-me': 0.03020547,
+                'kc_id_2_set-me': 0.0302166,
+                'kc_id_3_set-me': 0.03020543,
+                'kc_id_4_set-me': 0.03020937,
+                'kc_id_5_set-me': 0.03021411,
+                'kc_id_6_set-me': 0.81875085,
+                'kc_id_7_set-me': 0.03019817,
+            }
+        )
+    )
+    @unpack
+    @override_settings(
+        LDA_MODEL=joblib.load(
+            os.path.join(os.path.dirname(__file__), 'lda_test.pkl')
+        )
+    )
+    @override_settings(
+        TFIDF_VECTORIZER=joblib.load(
+            os.path.join(os.path.dirname(__file__), 'tfidf_vectorizer_test.pkl')
+        )
+    )
+    @override_settings(
+        GROUP_KCS=[
+            'kc_id_1_set-me', 'kc_id_2_set-me', 'kc_id_3_set-me',
+            'kc_id_4_set-me', 'kc_id_5_set-me', 'kc_id_6_set-me',
+            'kc_id_7_set-me'
+        ]
+    )
+    def test_calculate_probabilities(self, answers, expected_probabiities):
```

Comment:
@tomaszgy Could you add a docstring to this test?

Reply:
👍